osv/ecosystems.py

```diff
@@ -604,6 +627,15 @@ def enumerate_versions(self,
     'Packagist': Packagist(),
     'PyPI': PyPI(),
     'RubyGems': RubyGems(),
+    # Ecosystems missing implementations
+    'Android': OrderingUnsupportedEcosystem(),
+    'Github Actions': OrderingUnsupportedEcosystem(),
+    'Linux': OrderingUnsupportedEcosystem(),
+    'OSS-Fuzz': OrderingUnsupportedEcosystem(),
+    'Pub': OrderingUnsupportedEcosystem(),
+    # Ecosystems handled specially
+    'Alpine': OrderingUnsupportedEcosystem(),
```

Comment:
Honestly, I'm not sure.

If these aren't included here, the ecosystems of just "Debian" and "Alpine" (without the release numbers) would be considered invalid and thus filtered, even though [the schema says the release number is optional](https://ossf.github.io/osv-schema/#affectedpackage-field).

I included them in the dictionary as OrderingUnsupported to preserve the current behaviour in that case, rather than filtering them out.

Reply:
Got it, thanks! 

Please add a comment explaining this above these two.

Something like:
```
# Alpine and Debian require a release version for enumeration, which is
# handled separately in get().
'Alpine': ...,
'Debian': ...,
```